ds3_java_cli/src/main/java/com/spectralogic/ds3cli/Arguments.java

```diff
@@ -176,6 +178,7 @@ public Arguments(final String[] args) throws BadArgumentException, ParseExceptio
         this.options.addOption(numberOfThreads);
         this.options.addOption(ignoreErrors);
         this.options.addOption(noFollowSymlinks);
+        this.options.addOption(followSymLinks);
```

Comment:
The noFollowSymlinks is still there for compatibility.  The default for followsymlinks was changed from true to false, which is why the followSymLinks option was added.